src/opera/test/pge/dist_s1/test_dist_s1_pge.py

```diff
@@ -284,6 +285,274 @@ def test_dist_s1_pge_input_validation(self):
             if os.path.exists('temp'):
                 shutil.rmtree('temp')
 
+    def test_dist_s1_pge_input_rtc_validations(self):
+        """Test the input RTC validation checks made by DistS1PreProcessorMixin."""
+        runconfig_path = join(self.data_dir, 'test_dist_s1_config.yaml')
+        test_runconfig_path = join(self.data_dir, 'invalid_dist_s1_runconfig.yaml')
+
+        with open(runconfig_path, 'r', encoding='utf-8') as stream:
+            runconfig_dict = yaml.safe_load(stream)
+
+        backup_runconfig = deepcopy(runconfig_dict)
+
+        try:
+            Path('temp').mkdir(parents=True, exist_ok=True)
```

Comment:
This test should already execute in a temporary directory that will be automatically deleted after the test completes (pass or fail), so I'm not sure you need this or the cleanup line at the end.